elasticsearch.md

```diff
@@ -1685,6 +1688,14 @@ group_search_filter: "(cn=*)" # basically no group filtering
 
 Caching can be configured per LDAP client \(see `ldap1`\) or per rule \(see `Accept requests from users in group team2 on index2` rule\)
 
+#### Circuit Breaker
+
+The LDAP connector is equipped by default with the circuit breaker functionality. The circuit breaker disables sending new requests to the server when it doesn't respond properly. After receiving configured number of failed responses in a row, circuit breaker blocks sending new requests by terminating them immediately with exception. After configured time, circuit breaker allows one request to pass. If it succeeds, CB goes back to normal operation. If not test request is sent again after configured time. 
+
+Circuit breaker could be adapted to specific needs using these configuration parameters:
+* `max_retries` is number of failed responses in a row which will trigger circuit breaker.
+* `reset_duration` defines how long circuit breaker will block requests until sending one test request. 
```

Comment:
.... how long the circuit breaker feature will block the incoming requests before starting to send one test request. to the LDAP server.